### PR TITLE
{tools}[fosscuda/2020b] torchtext v0.10.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by dithwick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/torchtext/torchtext-0.10.0-fosscuda-2020b-PyTorch-1.9.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'torchtext'
+version = '0.10.0'
+local_pytorch_version = '1.9.0'
+local_python_suffix = '-Python-%(pyver)s'
```
Comment (bedroge) [inline easybuild/easyconfigs/t/torchtext/torchtext-0.10.0-fosscuda-2020b-PyTorch-1.9.0.eb:6]: Looks like this variable/line is redundant, now that the Python version has been dropped from the `versionsuffix`.
```suggestion
```

**Comment by dithwick** [comment]
Thanks @bedroge, that change made perfect sense.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/303e677222db7d9b0912e9116648ee26 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @dithwick!

**Pull request merged**